src/manifest.ts

```diff
@@ -118,8 +118,10 @@ export class Manifest {
 
             switch (suffix) {
                 case 'rust-stable':
```

Comment:
I hate this duplication, but it is the cleanest way I can think of to satisfy ts compiler.

Reply:
we can improve the situation later when needed. thank you and sorry for taking too long

Reply:
No worries :)